src/StatusQ/Controls/Validators/StatusValidator.qml

```diff
@@ -1,10 +1,12 @@
 import QtQuick 2.13
+import StatusQ.Controls 0.1
 
 QtObject {
     id: statusValidator
 
     property string name: ""
     property string errorMessage: "invalid input"
+    property StatusInput input
```

Comment:
This prevents the entire app from running 😱  I wonder if it's some kind of circular dependency, but nothing was reported in the output. Took a while to figure that one out.

Reply:
Oh damn! I've actually noticed that too. Made this change last minute without testing it 🙈 
Then realised that Status wasn't rendering a window any more.

Let's change this back to `var` then